src/robin_dict.jl

```diff
@@ -1,6 +1,6 @@
 import Base: setindex!, sizehint!, empty!, isempty, length, getindex, getkey, haskey, iterate, @propagate_inbounds, pop!, delete!, get, isbitstype
 
-const LOAD_FACTOR = 0.90
+const LOAD_FACTOR = 0.80
```

Comment:
Can this have a comment,
also should be `ROBIN_DICT_LOAD_FACTOR`

Reply:
Sure! That's the load factor I'm trying to achieve, but I can't seem to get close to that as of now!